src/index.js

```diff
@@ -0,0 +1,89 @@
+// import './style.css';
+
+const test = /[0-9]/g;
+const _ = require('lodash');
+const { printWeather } = require('./components/printComponents');
+const { setLocal, printHistory } = require('./components/history');
+
+function HistoryObject(data) {
+  this.location = `${data.location.country}, ${data.location.region}`;
+  this.temperature = data.current.temperature;
+  this.windDir = data.current.wind_dir;
+  this.windSpeed = data.current.wind_speed;
+  const icon = data.current.weather_icons[0];
+  this.icon = icon;
+  this.time = data.location.localtime.slice(11, 16);
+  this.pressure = data.current.pressure;
+  this.feelslike = data.current.feelslike;
+  this.discription = data.current.weather_descriptions;
+}
```

Comment:
```suggestion
const getHistoryObject = (data) => {
  const icon = data.current.weather_icons[0];
  
  return {
    location: `${data.location.country}, ${data.location.region}`,
    temperature: data.current.temperature,
    windDir: data.current.wind_dir,
    windSpeed: data.current.wind_speed,
    icon
    time: data.location.localtime.slice(11, 16),
    pressure: data.current.pressure,
    feelslike: data.current.feelslike,
    discription: data.current.weather_descriptions,
  }
};
```

чтобы вернуть объект не обязательно создавать функцию конструктор